common/service/pldm/pldm_firmware_update.h

```diff
@@ -226,15 +226,19 @@ enum pldm_component_classification_values {
  */
 enum pldm_firmware_update_descriptor_types {
 	PLDM_FWUP_IANA_ENTERPRISE_ID = 0x0001,
+	PLDM_PCI_DEVICE_ID = 0x0100,
 	PLDM_FWUP_VENDOR_DEFINED = 0xFFFF
 };
 
 /** @brief Descriptor types length defined in PLDM firmware update specification
  *  DSP0267 Table 7 – Descriptor identifier table
  */
-enum pldm_firmware_update_descriptor_types_length { PLDM_FWUP_IANA_ENTERPRISE_ID_LENGTH = 4 };
+enum pldm_firmware_update_descriptor_types_length {
+    PLDM_FWUP_IANA_ENTERPRISE_ID_LENGTH = 4,
+    PLDM_PCI_DEVICE_ID_LENGTH = 2
+ };
 
-/**
+/**pldm_firmware_update_descriptor_types_length
```

Comment:
Why did this comment get added after the the enum definition?  This belongs on line 233.

Reply:
Done